pkg/operator/ceph/file/mds/filesystem.go

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
Were filesystem.go and filesystem_test.go intended to be moved to the mds folder? Looks like these files are in both places now.

Reply:
That should have stayed in the `file` directory. Not sure how that happened.